plugins/oauth/hooks.js

```diff
@@ -14,7 +14,7 @@ import CookieUtils from "#SRC/js/utils/CookieUtils";
 import RouterUtil from "#SRC/js/utils/RouterUtil";
 import UsersPage from "#SRC/js/pages/system/UsersPage";
 
-import AuthenticatedUserAccountDropdown from "./components/AuthenticatedUserAccountDropdown";
+import AuthenticatedClusterDropdown from "./components/AuthenticatedClusterDropdown";
```

Comment:
The changes in these file are therefore not needed.

Reply:
Discussed offline.. we need the file.